src/resources/postcss/tribe-common-admin.pcss

```diff
@@ -90,9 +91,9 @@ td.tribe_message {
 /* = Plugin Screen
 =============================================*/
 table.plugins {
-
 	.tribe-plugin-update-message {
-		background: #d54e21; /* taken from color scheme in list-tables.css */
+		background: #d54e21;
+		/* taken from color scheme in list-tables.css */
```

Comment:
another auto-listing change, move this to line 95

Reply:
Should be fixed